docker/notebook/start.bash

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+echo "Setup Jupyter notebook connecting to $SQLFLOW_SERVER ..."
+jupyter notebook --ip=0.0.0.0 --port=8888 --allow-root --NotebookApp.token=''
```

Comment:
If it is a one-line command, how about we don't introduce this script file; instead, we write the following in the `docker/jupyter/Dockerfile`?

```dockerfile
CMD ["Jupiter", "notebook", "--ip=0.0.0.0", ..]
```